modules/references/tests/server/reference-create.server.routes.tests.js

```diff
@@ -1,14 +1,15 @@
 'use strict';
 
 var should = require('should'),
+    _ = require('lodash'),
     request = require('supertest'),
     async = require('async'),
     path = require('path'),
     sinon = require('sinon'),
     mongoose = require('mongoose'),
-    User = mongoose.model('User'),
     Reference = mongoose.model('Reference'),
     testutils = require(path.resolve('./testutils/server.testutil')),
+    utils = require('./utils'),
```

Comment:
the ./utils file could be moved to `./testutils/server.data.testutil.js` or somewhere similar.